python/cudf/cudf/core/column_accessor.py

```diff
@@ -341,7 +341,32 @@ def _pad_key(self, key: Any, pad_value="") -> Any:
             key = (key,)
         return key + (pad_value,) * (self.nlevels - len(key))
 
-
+    def replace_level_values(self, mapping: dict, level: int or None):
```

Comment:
We should use underscore in front of the internal function names, assuming it is one.